scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -419,9 +419,16 @@
 )
 
 # define helpers
-muon_prefiring_helper, muon_prefiring_helper_stat, muon_prefiring_helper_syst = (
-    muon_prefiring.make_muon_prefiring_helpers(era=era)
-)
+if era == "2016PostVFP":
```

Comment:
Can this change be made in make_muon_prefiring_helpers instead? As we pass the era to this function I think it should go there also to reduce duplication of code in other histmakers

Reply:
It can, but the point is that for 2018 (and maybe also 2017) we don't need helpers for the prefiring, we don't even have the histograms with the corrections, and currently the code just copies those for 2016.

Reply:
ok yes, this makes sense, I'll fix that